migrations/00016_index_on_refresh_tokens.sql

```diff
@@ -0,0 +1,4 @@
+SET ROLE postgres;
```

Comment:
this isn't needed and may actually break under some setups

Reply:
updated